SmartPlaces.Facilities/lib/IngestionManager/src/IngestionProcessorBase.cs

```diff
@@ -41,11 +42,13 @@ public abstract class IngestionProcessorBase<TOptions> : IGraphIngestionProcesso
         /// <param name="inputGraphManager">Manager for the input data graph that this ingestion processor parses.</param>
         /// <param name="ontologyMappingManager">Manager mapping between ontologies used by the input and output graphs.</param>
         /// <param name="outputGraphManager">Manager for the output data graph that this ingestion processor writes to.</param>
+        /// <param name="graphNamingManager">Manager for the naming of the elements of the graph.</param>
         /// <param name="telemetryClient">Application Insights telemetry client for remote metrics tracking.</param>
         protected IngestionProcessorBase(ILogger<IngestionProcessorBase<TOptions>> logger,
                                         IInputGraphManager inputGraphManager,
                                         IOntologyMappingManager ontologyMappingManager,
                                         IOutputGraphManager outputGraphManager,
+                                        IGraphNamingManager graphNamingManager,
```

Comment:
Should there be an update to the `IServiceCollection` extension methods with a default value?

Reply:
Done! Good catch... as long as we can override that later...